jetty-core/jetty-websocket/jetty-websocket-jetty-common/src/main/java/org/eclipse/jetty/websocket/common/JettyWebSocketFrame.java

```diff
@@ -20,10 +20,18 @@
 public class JettyWebSocketFrame implements org.eclipse.jetty.websocket.api.Frame
 {
     private final Frame frame;
+    private final byte effectiveOpCode;
 
+    @Deprecated
```

Comment:
With this deprecation there would be no alternative, is that intended?
If so, please add javadocs with `@deprecated` saying that.

Reply:
Well the alternative is the package private `JettyWebSocketFrame(Frame frame, byte effectiveOpCode)`.

This is really an internal implementation class translating a websocket core frame to a Jetty API frame.
So no reason a user should be using this class.

I'll add some javadoc though.

Reply:
@lachlan-roberts please also add `forRemoval=true` and `since="12.1.0".